src/uct/ib/rc/verbs/rc_verbs_impl.h

```diff
@@ -119,7 +120,18 @@ uct_rc_verbs_iface_poll_rx_common(uct_rc_verbs_iface_t *iface)
         goto out;
     }
 
-    UCT_IB_IFACE_VERBS_FOREACH_RXWQE(&iface->super.super, i, hdr, wc, num_wcs) {
+    for (i = 0; i < num_wcs; i++) {
+        desc = (uct_ib_iface_recv_desc_t *)(uintptr_t)wc[i].wr_id;
+        hdr = (uct_rc_hdr_t *)uct_ib_iface_recv_desc_hdr(&iface->super.super, desc);
+        if (ucs_unlikely(wc[i].status != IBV_WC_SUCCESS)) {
+            if (wc[i].status == IBV_WC_REM_ABORT_ERR &&
+                wc[i].vendor_err == UCT_IB_MLX5_CQE_VENDOR_SYND_PSN) {
```

Comment:
1. use ( )
2. we cannot use mlx5 from verbs code, just check for ABORT_ERR without syndrome